othdrplot/high_density_region_algorithm.py

```diff
@@ -3,7 +3,9 @@
 """
 Component to create HighDensityRegionAlgorithm.
 """
+from itertools import combinations_with_replacement
```

Comment:
This seems an unnecessary dependence to me: see the comment on the two `for` loops.